src/applications/vaos/utils/error.js

```diff
@@ -79,3 +79,15 @@ export function has400LevelError(error) {
 export function has409LevelError(error) {
   return getErrorCodes(error).some(code => code.startsWith('VAOS_409'));
 }
+
+export function has404AppointmentIdError(resp) {
```

Comment:
nit: I don't think the name of the function matches the actual behavior after changing it in the past few commits. Maybe something like `get404AppointmentIdError` or `resolve404AppointmentIdError`?

Reply:
Thanks for the feedback. I renamed it to `get404AppointmentIdError`